### PR TITLE
feat: accounts slugs

### PR DESCRIPTION
**Pull request by domenicsim1**

### CODE REVIEW

File: pkg/cmd/account/aws/create/create_test.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	const envID = "Env-1"
+	_ = fixtures.NewSpace(spaceID, "testspace")
+	_ = fixtures.NewEnvironment(spaceID, envID, "testenv")
+	api, qa := testutil.NewMockServerAndAsker()
+	out := &bytes.Buffer{}
+
+	opts := &create.CreateOptions{
+		CreateFlags: create.NewCreateFlags(),
+	}
+	opts.Space = spaceID
```
Comment (borland) [inline pkg/cmd/account/aws/create/create_test.go:106]: As above, trivial, but maybe nicer if these are assigned inside the `{ }` for CreateOptiosn

File: pkg/cmd/account/azure/create/create_test.go [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+	_ = qa.ExpectQuestion(t, &survey.Input{
+		Message: "Tenant ID",
+		Help:    "Your Azure Active Directory Tenant ID. This is a GUID in the format xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.",
+	}).AnswerWith("d2486c05-0cac-4d54-a91e-654043036f32")
+
+	_ = qa.ExpectQuestion(t, &survey.Input{
+		Message: "Application ID",
+		Help:    "Your Azure Active Directory Tenant ID. This is a GUID in the format xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.",
+	}).AnswerWith("d2486c05-0cac-4d54-a91e-654043036f33")
```
Comment (borland) [inline pkg/cmd/account/azure/create/create_test.go:80]: Tip: If you have some validation here (e.g. a validator that it's a GUID of the correct format) then it's very easy to throw a validation check into the middle of the test here.

```
q = qa.ExpectQuestion(t, &survey.Input { ... )

validationErr := q.AnswerWith("ddddd")
assert.EqualError(t, validationErr, "should be a guid format or something")

validationErr := q.AnswerWith("<proper guid>")
assert.Nil(t, validationErr)
```

File: pkg/cmd/account/ssh/create/create_test.go [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+	"github.com/stretchr/testify/assert"
+)
+
+var serverUrl, _ = url.Parse("http://server")
+
+const placeholderApiKey = "API-XXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+var rootResource = testutil.NewRootResource()
+
+func TestGCPAccountCreatePropmtMissing(t *testing.T) {
```
Comment (borland) [inline pkg/cmd/account/ssh/create/create_test.go:30]: Prompt

**Review by borland — CHANGES_REQUESTED** [review]

File: pkg/cmd/account/aws/list/list.go [hunk 4]
```diff
@@ -47,19 +47,21 @@ func listAwsAccounts(client *client.Client, cmd *cobra.Command) error {
 			acc := item.(*accounts.AmazonWebServicesAccount)
 			return &struct {
 				Id        string
+				Slug      string
```
[the remark this answers is not in the document]
Comment (slewis74) [reply pkg/cmd/account/aws/list/list.go:50]: I'd assume it'll just display a blank, which I think is fine 👍

**Review by slewis74 — APPROVED** [review]
Overall looking 👍 

Per @borland 's comment, a few of the tests have `Propmt` in the name, rather than `Prompt`, which would be worth fixing.

**Review by borland — APPROVED** [review]

**Pull request merged**